File: templates/clojure.midje/%kata_file%.clj
; Adapt the code to your code kata %kata_file%.

(ns %kata_file%
  (:use [midje.sweet]))

(use 'clojure.test)

(defn %kata_file% []
  "fixme")

(fact (%kata_file%) => "foo")

(run-tests)

File: templates/haskell.hunit/%kata_file%.hs
-- Adapt the code to your code kata %kata_file%.

import Test.HUnit

foo :: String
foo = "fixme"


test1 = TestCase (do assertEqual "" "foo" (foo))

tests :: Test
tests = TestList [TestLabel "test1" test1]

main :: IO Counts
main = do runTestTT tests

